from exceptions import ingrese_numero, NumeroDebeSerPositivo

def main():
    return
